client/pages/event-details.jsx

```diff
@@ -0,0 +1,115 @@
+import { Card, Grid, Typography, CardContent, Container } from '@material-ui/core';
+import { makeStyles } from '@material-ui/styles';
+import React, { useEffect, useState } from 'react';
+import BackButton from '../components/back-button';
+import Map from '../components/map';
+import timestampConversion from '../lib/date-and-time-conversion';
+
+const useStyles = makeStyles(theme => (
+  {
+    heading: {
+      fontSize: '1.7rem',
+      fontWeight: '300',
+      marginTop: '0.35em'
+    },
+
+    bold: {
+      fontWeight: '700'
+    },
+
+    card: {
+      height: '100%'
+    },
+
+    mapHeight: {
+      minHeight: '20rem'
+    },
+
+    spacingbutton: {
+      marginTop: theme.spacing(2),
+      marginBottom: theme.spacing(2)
+    },
+
+    spacingtext: {
+      marginBottom: theme.spacing(2)
+    }
+  }
+));
+
+export default function EventDetails(props) {
+  const [event, setEvent] = useState(null);
+  const [center, setCenter] = useState(null);
+  const [marker, setMarker] = useState(null);
+
+  const handleLoad = () => {
+    const coords = event.coords.destinationCoords;
+    setCenter(coords);
+    setMarker(coords);
+  };
+
+  useEffect(() => {
+    fetch(`api/events/${props.eventId}`)
+      .then(res => res.json())
+      .then(result => {
+        setEvent(result);
+      });
+  }, []);
+
+  const classes = useStyles();
+
+  const render = () => {
```

Comment:
Don't make a function like this, just return the elements 😂

Reply:
OOPS LOL